Maestro/subscriptions.json

```diff
@@ -1818,6 +1819,22 @@
         }
       }
     },
+    // Automate opening PRs to merge dotnet org repos' release/3.1 changes into master branches
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/wpf/blob/release/3.1/**/*"
```

Comment:
This suggests you're going with the approach of merging any change into the oldest applicable branch first, then letting it flow to `master` eventually. In the Core repos changes are instead merged into `master` first, then ported into the oldest applicable branch. Repos do it either way--as far as I'm aware it's a matter of opinion. Wondering if it's a conscious choice for WPF in particular.

Reply:
We would want commits to be merged into simultaneously into `master `as well as `release/* `branches as a best-practice. This is a back-stop to guarantee "forward porting" of fixes. For e.g., suppose we take a small point-fix into `release/3.0` directly that for some reason was missed being committed to `master `directly, this rule would ensure that it would flow into `release/3.1` and also `master`. OTOH, if a fix was made into `master`, then cherry-picked into `release/3.0`, then it would simply flow into `release/3.1`(useful), and and ensuing merge from `release/3.1` -> `master `would be empty (noise - but no harm done). 

Since the churn in `release/*` branches is generally low, it seems like a small price to pay (dealing with occasional empty merge PR's from `release/3.1 -> master`) to guarantee that we don't forget to forward-port changes inadvertently.

Reply:
Seems like a reasonable approach, this can cause conflicts when some commit is ported from `master` to `3.1` after the fact, if some other commit has changed the same line. Someone has to go figure out the conflict. Ran an example to prove it to myself, in case the scenario isn't clear:

![image](https://user-images.githubusercontent.com/12819531/64441794-2c640380-d094-11e9-85c4-072978e5ce86.png)

`1` and `2` just change the only line in a file to that number. The conflict is:

```
<<<<<<< HEAD
2
=======
1
>>>>>>> 3.1
```

Not sure if this is something you haven't considered. Being able to tell if a change is properly everywhere based on whether it's contained in certain Git trees would certainly be useful.

Reply:
thx - this is a good caveat to remember. I still think I'd like to onboard it this way. If it turns out to be a hassle, we can always change/weak this process, right? if that's ok, then lets go ahead and merge this.